### PR TITLE
add final touch and pagination

### PR DESCRIPTION
**Pull request by Monaeid2001**

### CODE REVIEW

File: Views/login_form.php
Comment (omarkhalil117) [inline Views/login_form.php:1]: keep the old condition there is no point on checking isset and isnot null we only need to know if there is a session[id] or not

File: .gitignore
Comment (omarkhalil117) [inline .gitignore:1]: why is it empty ??

File: Components/navbar.php
Comment (omarkhalil117) [inline Components/navbar.php:1]: where is the navbar ??

File: Components/product_card.php
Comment (omarkhalil117) [inline Components/product_card.php:1]: ???

File: Views/login_form.php
Comment (Monaeid2001) [reply Views/login_form.php:1]: Okay

**Pull request closed without merging**